app/src/main/java/nerd/tuxmobil/fahrplan/congress/models/Lecture.java

```diff
@@ -22,8 +22,10 @@
 
     public String room;
 
-    // The roomIndex value is not something the UI should be concerned about. Remove this once all
-    // code is switched over from using LegacyLectureData to using ScheduleData.
+    // This value contributes to how rooms are ordered in the user interface. But it is NOT an index
+    // into the list of rooms displayed in the UI. Due to problems in the way a roomIndex value is
+    // generated there is currently no guarantee that two different rooms have different roomIndex
+    // values.
```

Comment:
- Please use /** syntax
- I still don't understand "not an index into the list". Maybe we can talk about that offline?
- Could you somewhere write down (github issue, TODO / FIXME in code) your suspicion about roomIndex creation, so that its not forgotten?

Reply:
Have you seen my comment? I still think this are valid suggestions.
I suggest to write "not an index *of the* list".
Do you know more about "problems in the way a roomIndex value is generated"? If so, please reference or explain them here. Otherwise that sentence has little value.

Reply:
Changed the text in 5e7a8fc070c5bc392a77a4998fa89869c67ead4d. 

It does not explain why the value should not be used, but apparently it's hard to communicate that in a few sentences. So rather than having an endless debate on what exactly the text should and should not contain I opted for this variant that leaves questions open instead. At some point people need to read the code anyway.